File: results/fine_tuning/test_output_files/question-82-pass-1.py
import math


def isPowerOfTwo(n):
    if (n == 1):
        return 1
    if (n == 0):
        return 0
    if (n & (n - 1)):
        return 0
    return 1


if (isPowerOfTwo(10)):
    print("Yes")
else:
    print("No")


if (isPowerOfTwo(16)):
    print("Yes")
else:
    print("No")


if (isPowerOfTwo(100)):
    print("Yes")
else:
    print("No")


if (isPowerOfTwo(1000)):
    print("Yes")
else:
    print("No")


if (isPowerOfTwo(1000000)):
    print("Yes")
else:
    print("No")


if (isPowerOfTwo(1000000000)):
    print("Yes")
else:
    print("No")


if (isPowerOfTwo(100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000

File: results/feedback_pipeline/cpp_results/cpp_files_after_feedback/question-82-pass-1.cpp
#include <bits/stdc++.h>
using namespace std;
bool isPowerOfTwo(int n) {
    return (n > 0) && (n & (n - 1)) == 0;
}
int main() {
    int n;
    cin >> n;
    if (isPowerOfTwo(n)) {
        cout << "Yes";
    } else {
        cout << "No";
    }
    return 0;
}